docs/linux-setup.md

```diff
@@ -0,0 +1,9 @@
+### Getting started on LibVLCSharp for Linux
+
+1. Install **[Mono](https://www.mono-project.com/download/stable/#download-lin)**. Make sure you can invoke it using `msbuild`.
+2. Install **libvlc**: 
+> `apt-get install libvlc-dev`. 
+
+##### `libvlc.so` and `libvlccore.so` need to be located at `/usr/lib` (probably symlinked).
```

Comment:
`##` are used for headings. use `*` insteads.

I'd remove `probably symlinked` as it's always a symlink (it's the way linux works)